src/docs/src/conf.py

```diff
@@ -29,9 +30,13 @@
 
 nitpicky = True
 
-# should be over-written using rebar-inherited settings
-version = "3.3"
-release = "3.3.0"
+# load version numbers from version.mk
+version_file = Path().absolute().joinpath('../../../version.mk').resolve()
```

Comment:
I'm seeing `black` complain about the single quotes around `'../../../version.mk'`. 

`make python-black-update` changes it to double quotes.